gateway-export-plugin/src/test/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/loader/ScheduledTaskLoaderTest.java

```diff
@@ -7,82 +7,68 @@
 package com.ca.apim.gateway.cagatewayexport.tasks.explode.loader;
 
 
-import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentTools;
-import com.ca.apim.gateway.cagatewayexport.tasks.explode.bundle.entity.ScheduledTaskEntity;
-import com.google.common.collect.ImmutableMap;
-import org.junit.jupiter.api.Test;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-
-import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BuilderUtils.buildAndAppendPropertiesElement;
-import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.createElementWithAttributesAndChildren;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.createElementWithChildren;
-import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.createElementWithTextContent;
-import static org.junit.jupiter.api.Assertions.*;
-
 class ScheduledTaskLoaderTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=122&issues=AWblzesRSgN4uCclhZKZ&open=AWblzesRSgN4uCclhZKZ)

[ISSUE_KEY]: AWblzesRSgN4uCclhZKZ